oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
Little gist that works on GCE

https://gist.github.com/elibixby/40c07cd47df15f7b0f1db0f575518058